runtime/makelib/targets.mk.win.inc.ftl

```diff
@@ -135,22 +134,35 @@ UMA_EXEFLAGS+=/INCREMENTAL:NO /NOLOGO /LARGEADDRESSAWARE
 #	/Zd Add line number info
 #	/Zi Add symbolic debug info
 #	/Gd Use C calls (i.e. prepend underscored to symbols)
-#
-ASFLAGS+=/c /Cp /W3 /nologo /safeseh /coff /Zm /Zd /Zi /Gd -DWIN32
+ASFLAGS+=/c /Cp /W3 /nologo /Zd /Zi -DWIN32 -D_WIN32 -DOMR_OS_WINDOWS
+<#if uma.spec.processor.x86>
+ASFLAGS+=/safeseh /coff /Zm /Gd
 <#elseif uma.spec.processor.amd64>
-ASFLAGS+=/c /Cp /W3 /nologo /Zd /Zi  -DWIN32 -DWIN64 -DJ9HAMMER
+ASFLAGS+=-DWIN64 -D_WIN64 -DJ9HAMMER
 </#if>
+
+CFLAGS+=-DWIN32 -D_WIN32 -DOMR_OS_WINDOWS
+CXXFLAGS+=-DWIN32 -D_WIN32 -DOMR_OS_WINDOWS
+ifdef USE_MINGW
+  MINGW_CXXFLAGS+=-DWIN32 -D_WIN32 -DOMR_OS_WINDOWS
+endif
+
 <#if uma.spec.processor.amd64>
-  <#if uma.spec.flags.build_newCompiler.enabled>
-    CFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DWIN32 -D_WIN32 -DJ9HAMMER
-    CXXFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DWIN32 -D_WIN32 -DJ9HAMMER
-  <#else>
-    CFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DWIN32 -D_WIN32 -DJ9HAMMER /FIPRE64PRA.H -Wp64
-    CXXFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DWIN32 -D_WIN32 -DJ9HAMMER /FIPRE64PRA.H -Wp64
+  CFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DJ9HAMMER
+  CXXFLAGS+=-D_AMD64_=1 -DWIN64 -D_WIN64 -DJ9HAMMER
+  <#if !uma.spec.flags.build_newCompiler.enabled>
```

Comment:
You can just delete the build_newCompiler check and options, its obsolete and no longer supported.

Reply:
It looks like it is enabled in our specs: 
```xml
buildspecs/j9.flags:159:        <flag id="build_newCompiler">
buildspecs/win_x86-64.spec:141:         <flag id="build_newCompiler" value="true"/>
buildspecs/win_x86-64_cmprssptrs.spec:137:              <flag id="build_newCompiler" value="true"/>
buildspecs/win_x86.spec:140:            <flag id="build_newCompiler" value="true"/>
```
I will delete this flag, but should I delete the compile options?

Reply:
Right sorry. We need the options, but we don't need to check the flag which is always true and can be removed.

Reply:
Right, makes sense. Thanks for clarifying!

Reply:
Woops I misread the code last night.  We *did* want to delete the compile options.  Fixed.